vk/server.go

```diff
@@ -100,12 +104,14 @@ func (s *Server) TestStart() error {
 	s.started.Store(true)
 
 	// mount the root set of routes before starting
-	s.router.Finalize()
```

Comment:
Do we need to keep a reference to the original router and if so are we certain it never needs to be finalized?

Reply:
the `Finalize` has a `sync.Once` in it, so even if we repeatedly call it, it will only ever run once. The decision for why this is so is with @cohix, and I don't know enough about the context of why it was built in such a way yet to answer this :)